src/Ocelot/Configuration/Repository/DiskFileConfigurationRepository.cs

```diff
@@ -14,21 +15,33 @@ namespace Ocelot.Configuration.Repository
     public class DiskFileConfigurationRepository : IFileConfigurationRepository
     {
         private readonly IOcelotConfigurationChangeTokenSource _changeTokenSource;
+        private readonly IOcelotCache<FileConfiguration> _cache;
         private readonly string _environmentFilePath;
         private readonly string _ocelotFilePath;
+        private readonly string _cacheKey;
         private static readonly object _lock = new();
         private const string ConfigurationFileName = "ocelot";
 
-        public DiskFileConfigurationRepository(IWebHostEnvironment hostingEnvironment, IOcelotConfigurationChangeTokenSource changeTokenSource)
+        public DiskFileConfigurationRepository(IWebHostEnvironment hostingEnvironment,
+            IOcelotConfigurationChangeTokenSource changeTokenSource, Cache.IOcelotCache<FileConfiguration> cache)
         {
             _changeTokenSource = changeTokenSource;
+            _cache = cache;
             _environmentFilePath = $"{AppContext.BaseDirectory}{ConfigurationFileName}{(string.IsNullOrEmpty(hostingEnvironment.EnvironmentName) ? string.Empty : ".")}{hostingEnvironment.EnvironmentName}.json";
+            _cacheKey = "InternalDiskFileConfigurationRepository";
```

Comment:
could be a constant

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1249/commits/d052f478d3586db3ec8c93351b7a73a96e78bffe ✔️